pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -388,6 +388,25 @@ def test_take_fill_value_with_timezone(self):
 
 
 class TestGetLoc:
+    def test_get_loc_key_unit_mismatch(self):
```

Comment:
Side-related. Should this work when the key is tz-aware and idx is tz-naive (and vise versa)?

Reply:
ATM it works with mismatched aware/naive, but that is deprecated.